File: CS/13_Conversion/ToPDFAWithMetadata.cs
using System;
using System.Windows.Forms;
using Spire.Pdf;
using Spire.Pdf.Conversion;
using Spire.Pdf.Interchange.Metadata;

namespace ToPDFAWithMetadata
{

	public partial class Form1 : Form
	{
        public Form1()
        {
            InitializeComponent();
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            string path = @"..\..\..\..\..\..\Data\ToPDFAWithMetadata.pdf";
            string pdfA = "ToPDFAWithMetadata_out.pdf";

            // Create an instance of the PdfStandardsConverter class
            PdfStandardsConverter converter = new PdfStandardsConverter(path);
            // Convert to PDFA format document to preserve XMP data
            converter.Options.PreserveAllowedMetadata = true;
            converter.ToPdfA1A(pdfA);

            //Launch the file
            DocumentViewer(pdfA);
        }
        private void DocumentViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

        private void btnClose_Click(object sender, EventArgs e)
        {
            Close();
        }

	}
}

File: VB/13_Conversion/ToPDFAWithMetadata.vb
Imports Spire.Pdf
Imports Spire.Pdf.Conversion
Imports Spire.Pdf.Interchange.Metadata

Namespace ToPDFAWithMetadata

	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			Dim path As String = "..\..\..\..\..\..\Data\ToPDFAWithMetadata.pdf"
			Dim pdfA As String = "ToPDFAWithMetadata_out.pdf"

			' Create an instance of the PdfStandardsConverter class
			Dim converter As New PdfStandardsConverter(path)
			
			' Convert to PDFA format document to preserve XMP data
			converter.Options.PreserveAllowedMetadata = True
			converter.ToPdfA1A(pdfA)

			'Launch the file
       DocumentViewer(pdfA)
		End Sub
		Private Sub DocumentViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub

		Private Sub btnClose_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnClose.Click
			Close()
		End Sub

	End Class
End Namespace
